pilot/pkg/proxy/envoy/v2/xds_test.go

```diff
@@ -106,7 +106,7 @@ func startEnvoy(t *testing.T) {
 		"NodeID":  nodeID,
 		"BaseDir": env.IstioSrc + "/tests/testdata/local",
 		// Same value used in the real template
-		"meta_json_str": fmt.Sprintf(`"BASE": "%s"`, env.IstioSrc+"/tests/testdata/local"),
+		"meta_json_str": fmt.Sprintf(`"BASE": "%s", ISTIO_PROXY_VERSION: 1.1.3`, env.IstioSrc+"/tests/testdata/local"),
```

Comment:
Why version is hardcoded? Does it work when the code evolves?

Reply:
What do you recommend? We don't have 1.1.3 as a constant anywhere. The proxy version is just the compatibility version (not directly tied to Istio version), so if we want to test features that need a newer version we can just increment the version. 

We only had to add the version here (and disable the test previously) because we added a new feature that required 1.1.3+ and didn't update the test

Reply:
make sense